client/src/components/settings/Portfolio.tsx

```diff
@@ -43,16 +49,18 @@ function createEmptyPortfolio() {
   };
 }
 
-function createFindById(id) {
-  return p => p.id === id;
+function createFindById(id: string) {
+  return (p: PortfolioValues) => p.id === id;
 }
 
 const mockEvent = {
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
   preventDefault() {}
```

Comment:
I'm confused with what the `{}` braces does in this line. Also this function is called below in line 110. I've pasted the respective function that calls it here.
```js
handleRemoveItem = (id: string) => {
    return this.setState(
      state => ({
        portfolio: state.portfolio.filter(p => p.id !== id)
      }),
      () => this.handleSubmit(mockEvent)
    );
  };
```

Reply:
I think this might be needed for testing, but I am unsure.

Reply:
I searched the whole repo, and the function isn't called anywhere.

Reply:
It is called here:
```
  handleRemoveItem = id => {
    return this.setState(
      state => ({
        portfolio: state.portfolio.filter(p => p.id !== id)
      }),
      () => this.handleSubmit(mockEvent)
    );
  };
```
And used here (to not throw an error:
```
handleSubmit = e => {
    e.preventDefault();
```

Reply:
Umm I pasted that same function above in my comment here and yea those two were the only places which rely on it.

Reply:
I must be tired...sorry.

Reply:
It's fine